### ISSUE TITLE
Accessibility: language not set

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#3073]
Language of page has not been set
 3.1.1 Language of Page
The webpage is missing a definition of the natural language.

Provide a 'lang' attribute for HTML documents or an xml:lang attribute for xhtml documents to indicate the natural language of the page.

Use one of the following techniques from WCAG 2:

H57 Using language attributes on the html element
https://www.w3.org/TR/WCAG20-TECHS/H57
These references are taken from the Web Content Accessibility Guidelines 2.
